tests_new/integration_tests/README.md

```diff
@@ -62,6 +66,11 @@ Currently **we support only Cognito based deployments** but support for any IdP
           "accountId": "...",
           "region": "eu-west-1"
         }
+      },
+      "dasbhoards": {
```

Comment:
nit: typo dasbhoards --> dashboards

Reply:
nit: in the README we are saying we use the session_env but then in the code we are using the persistent environment

Reply:
sorry forgot to push typo fix - pushed now

we use `session_env1` everywhere now and not `persistent_env1`